config/Migrations/20201223141106_SentencesReplaceHashWithUniqueKey.php

```diff
@@ -0,0 +1,25 @@
+<?php
+use Migrations\AbstractMigration;
+
+class SentencesReplaceHashWithUniqueKey extends AbstractMigration
+{
+    /**
+     * Change Method.
+     *
+     * More information on this method is available here:
+     * https://book.cakephp.org/phinx/0/en/migrations.html#the-change-method
+     * @return void
+     */
+    public function change()
+    {
+        $table = $this->table('sentences');
+        $table->addIndex(['text', 'lang'], [
+            'unique' => true,
+            'name' => 'unique_text_lang'
```

Comment:
Sorry but I forgot that we can't use a unique index.

Whenever someone changes a sentence so that this sentence becomes a duplicate of another already existing sentence we will allow these duplicate pair to exist in the database. The two sentences will be merged by [Horus](https://github.com/Tatoeba/horus), a bot that runs every 15 minutes.

Reply:
@AndiPersti Done!